### ISSUE TITLE
Accept short yt url format

### ISSUE THREAD

**Issue by gwpl** [zaarheed/transvribe#5]
Could you consider supporting short yt format like:

* https://youtube.com/shorts/_7A65xq9RKI

?

if by hand I change it to `youtu.be/$hash` it alredy works:

* https://youtu.be/_7A65xq9RKI

**Comment by zaarheed** [comment]
Yes this should be a pretty straight-forward change. It currently doesn’t accommodate YouTube shorts because I never considered them. 

I’ll have this change live in a few days unless someone beats me to it!
